provider/database.py

```diff
@@ -18,7 +18,8 @@
 from cloudant.result import Result
 
 class Database:
-    dbname = "ow_kafka_triggers"
+    db_prefix = os.environ['DB_PREFIX']
```

Comment:
Need to inform the user what this environment variable should be set to. This means we need to update the README to inform others how to build, deploy, and run the service.

What are the advantages of using an environment variable here instead of passing in an argument to the Docker run command for set the db_prefix?